plugin/trino-atop/src/main/java/io/trino/plugin/atop/AtopConnectorFactory.java

```diff
@@ -68,7 +69,9 @@ public Connector create(String catalogName, Map<String, String> requiredConfig,
                             AtopConnectorConfig.class,
                             config -> config.getSecurity() == AtopSecurity.FILE,
                             binder -> {
-                                binder.install(new FileBasedAccessControlModule());
+                                FileBasedAccessControlModule module = new FileBasedAccessControlModule();
+                                module.setConfigurationFactory(new ConfigurationFactory(requiredConfig));
```

Comment:
Why do we need to inject the ConfigurationFactory manually ? FileBasedAccessControlModule extends `AbstractConfigurationAwareModule` then we might not need it.

Reply:
refactored